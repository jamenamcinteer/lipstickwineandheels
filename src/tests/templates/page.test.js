import React from 'react'
import { shallow } from 'enzyme'
import PageTemplate from '../../templates/page'
import data from '../fixtures/data'

test('should render page template correctly', () => {
  const wrapper = shallow(<PageTemplate data={data.data} />)
  expect(wrapper).toMatchSnapshot()
})
