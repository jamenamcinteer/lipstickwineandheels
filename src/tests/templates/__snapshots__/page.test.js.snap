// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page template correctly 1`] = `
<PageTransition>
  <div
    style={
      Object {
        "background": "#fff",
      }
    }
  >
    <HelmetWrapper
      defer={true}
      encodeSpecialCharacters={true}
      title="About | Lipstick, Wine, and Heels"
    >
      <html
        lang="en"
      />
      <meta
        content="This page is all about me"
        name="description"
      />
    </HelmetWrapper>
    <div
      className="heroContainer"
    >
      <div
        className="hero"
        style={
          Object {
            "backgroundImage": "url(/reference/to/some/image.jpg)",
          }
        }
      />
    </div>
    <div
      className="pageContainer"
    >
      <h1
        className="section-headline"
      >
        About
      </h1>
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<p>Learn all about me...</p>",
          }
        }
      />
    </div>
  </div>
</PageTransition>
`;
